lib/fast_rcnn/train.py

```diff
@@ -15,7 +15,11 @@
 import os
 
 from caffe.proto import caffe_pb2
-import google.protobuf as pb2
+
+# Importing google.protobuf and accessing text_format from it doesn't work,
+# but this does.
+import google.protobuf.text_format as pb2_text_format
```

Comment:
I made this change when I found the previous import of the library wasn't working properly.  As you say, it appears to be an issue with the particular version of protobuf I was running it with.  If the import in in this commit works with protobuf 2.6.1, I don't see the harm in keeping both your commit and this change.